.env.template

```diff
@@ -11,3 +11,7 @@ PJM_API_KEY=
 ERCOT_API_USERNAME=
 ERCOT_API_PASSWORD=
 ERCOT_API_SUBSCRIPTION_KEY=
+
+# Sign up here and subscribe to the pricing product
+# https://data-exchange.misoenergy.org/apis
```

Comment:
The Sign Up link here just redirects to the Sign In page so I don't actually know how to create a new account.